pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -367,20 +346,36 @@ def _apply_lightning_controlled(self, sim, operation):
             Returns:
                 array[complex]: the output state tensor
             """
-            basename = "PauliX" if operation.name == "MultiControlledX" else operation.base.name
-            if basename == "Identity":
-                return
-            method = getattr(sim, f"{basename}", None)
             control_wires = self.wires.indices(operation.control_wires)
-            control_values = (
-                [bool(int(i)) for i in operation.hyperparameters["control_values"]]
-                if operation.name == "MultiControlledX"
-                else operation.control_values
-            )
+
             if operation.name == "MultiControlledX":
+                basename = "PauliX"
+                control_values = [bool(int(i)) for i in operation.hyperparameters["control_values"]]
                 target_wires = list(set(self.wires.indices(operation.wires)) - set(control_wires))
```

Comment:
This special case is not necessary anymore. The `MultiControlledX` now inherits from `Controlled`, you can access its base like `operation.base`, its control values like `operation.control_values` just like you would with any other operator.